wire-ios-sync-engine/Source/Synchronization/Strategies/SearchUserImageStrategy.swift

```diff
@@ -32,17 +32,24 @@ public class SearchUserImageStrategy: AbstractRequestStrategy {
     fileprivate var requestedPreviewAssetsInProgress: Set<UUID> = Set()
     fileprivate var requestedCompleteAssetsInProgress: Set<UUID> = Set()
 
-    fileprivate var observers: [Any] = []
+    fileprivate var observers: [any NSObjectProtocol] = []
+
+    private let searchUsersCache: SearchUsersCache?
```

Comment:
is it supposed to be optional?

Reply:
In this case I tried not to change too much of the existing parts, but after our call I could do the dependency object non-optional: 985f0728dce45937f14b57eee37825510e7d2c7a